### DISCUSSION TITLE
AttributeError: module 'pypdfium2' has no attribute 'render_pdf_topil'

### DISCUSSION THREAD

**Discussion by zxnga** [mindee/doctr#952] [Q&A] [answered]
Hello everyone,

I came across the project and I wanted to try it out.
I tried to run the example that goes with the installation steps but unfortunately I get this `AttributeError: module 'pypdfium2' has no attribute 'render_pdf_topil'` error which I don't know how to solve when running :

`doc = DocumentFile.from_pdf("<path_to_my_pdf_file>")`

At first I thought I had my install wrong so I ran the example notebooks in colab and got the same error.
Would you have some clues about what I should do ?

I tried to downgrade the pypdfium2 module but it did not work either.

Thank you!

**Comment by felixdittrich92** [comment, marked as answer]
Hi @TheoZan, 
we have this on track and will fix this soon for the meanwhile could you check if it work with pypdfium2==1.0.0 ? :)

#947

**Reply by zxnga** [reply]
Hey @felixdittrich92,

Thank you and sorry the late answer. 
I tried with pypdfium2==1.0.0, and it works juste fine !!

Thank you very much for that :)
